about.php

```diff
@@ -104,7 +97,14 @@ classmates.
 			00001010 Web Development Street&nbsp; |&nbsp; Buffalo , NY 14220&nbsp; |&nbsp;
 		</b>
 	</div>
-	</footer>
+	<div id="socialMedia">
+		<a href="http://www.instagram.com/" class="btn btn-default" target="_blank">
+			<img src="images\instaIcon.jpg" alt="Instagram"></a>
+		<a href="http://www.facebook.com/" class="btn btn-default" target="_blank">
+			<img src="images\faceIcon.jpg" alt="facebook"></a>
+		<a href="http://www.twitter.com/" class="btn btn-default" target="_blank">
+			<img src="images\twitterIcon.jpg" alt="twitter"></a>
```

Comment:
this is still doing the weird thing with the `</a>` at the end rather than its own line